src/Maestro/tests/Scenarios/goal.ps1

```diff
@@ -0,0 +1,55 @@
+param(
+    [string]$maestroInstallation,
+    [string]$darcVersion,
+    [string]$maestroBearerToken,
+    [string]$githubPAT,
+    [string]$azdoPAT
+)
+
+try {
+    Write-Host
+    Write-Host "Darc/Maestro Goal tests"
+    Write-Host
+
+    # Import common tooling and prep for tests
+    . $PSScriptRoot/common.ps1
+
+    # Test channel name
+    $newChannelName = Get-Random
+    $newDefinitionId = Get-Random
+    $newMinutes = Get-Random
+
+    # Add a new channel
+    Write-Host "Creating channel '$newChannelName'"
+    Darc-Add-Channel -channelName $newChannelName -classification 'test'
+
+    # Add a new Goal for a Definition in a Channel 
+    Write-Host "Creating goal" 
+    Darc-Set-Goal -channel $newChannelName -definitionId $newDefinitionId -minutes $newMinutes
+
+    # Get the goal and make sure it's there
+    Write-Host "Checking that the goal is added"
```

Comment:
Can we update the output messages to match how we do it in the other scripts? So, here:

```
Write-Host "Checking that goal $newMinutes for '$newDefinitionId' on channel '$newChannelName' exists"
```

and below:

```
throw "Cannot find goal $newMinutes for definition '$newDefinitionId' on channel '$newChannelName'"
```

and then also be more descriptive on the else

Reply:
sure updated logs and exception msgs,